bin/check-log.rb

```diff
@@ -33,7 +33,7 @@
 require 'fileutils'
 
 class CheckLog < Sensu::Plugin::Check::CLI
-  BASE_DIR = '/var/cache/check-log'.freeze
+  BASE_DIR = '/var/cache/sensu/check-log'.freeze
```

Comment:
Is there a reason to not make this an option and default to this? I tend to dislike configuration that is not exposed.